egov-indexer/project-task-indexer.yml

```diff
@@ -38,4 +38,4 @@ ServiceMaps:
                 outJsonPath: $.Data.locality
               - inJsonPath: $.village
                 outJsonPath: $.Data.village
```

Comment:
@amitverma-egov please add auditDetails.createdTime also as syncedTime into user-sync-index-v1 wherever this is applicable in all indexer files and also the timestamps for both clientAuditDetails and auditDetails.

@saiprakash-eGov

Reply:
@bhanuprakash-egov  I have made the changes as requested ,please review the them